### ISSUE TITLE
issue with running tracer for a cgc binary

### ISSUE THREAD

**Issue by benquike** [angr/tracer#29]
When running Tracer with  this binary,  [KPRCA_00034_patched](https://github.com/benquike/cgc-challenge-corpus/blob/master/KPRCA_00034/bin/KPRCA_00034_patched?raw=true), with this file 
[id:000000,sig:11,src:000157,op:havoc,rep:4.zip](https://github.com/angr/tracer/files/1143731/id.000000.sig.11.src.000157.op.havoc.rep.4.zip), the following exception happpens.

```
IndexError: list index out of range
```

And the reason is that when the tracer runs that binary, `shellphish-qemu-cgc-tracer` exits with the following error.

```
Error while loading /export/users/cgc-vm_fuzz/cgc-challenge-corpus/KPRCA_00034/bin/KPRCA_00034_patched: Operation not permitted
```

**Comment by salls** [comment]
Can you check the permissions on the binary? make sure it's executable

**Comment by benquike** [comment]
Yes, it has `x` permission

**Comment by benquike** [comment]
I found that it is caused by the `-magicdump` option.
If the file specified by the argument exists, this issue will happen.

**Comment by benquike** [comment]
I finally figured out what was wrong.

**Comment by zardus** [comment]
Would you mind writing a quick blurb about the cause to help future
sufferers?

On Jul 20, 2017 5:00 PM, "Hui Peng" <notifications@github.com> wrote:

Closed #29 <https://github.com/angr/tracer/issues/29>.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/angr/tracer/issues/29#event-1173077978>, or mute the
thread

**Comment by benquike** [comment]
@zardus  It was that the directory for storing the qemu trace logs ran out of space. 
Because previous runs of tracer exited in an code path in which the trace log was not
removed and the logs accumulated .... finally all the space was eated up
